src/bscPlugin/codeActions/CodeActionsProcessor.spec.ts

```diff
@@ -225,6 +225,51 @@ describe('CodeActionsProcessor', () => {
                 `import "pkg:/source/Animals.bs"`
             ]);
         });
+
+        it('suggests callfunc operator refactor', () => {
+            //define the function in two files
+            const file = program.addOrReplaceFile('components/main.bs', `
+                sub doSomething()
+                    someComponent.callfunc("doThing")
```

Comment:
nobody would actually have this code, coz it would not work (it woudl do nothing at run time).

It would have to be `someComponent.callfunc("doThing", invalid)`, which (imho) should become `someComponent@.doThing()` (which is semantically equivalent).

Reply:
While I understand the "standard", I've seen callfunc in the wild with no second parameter. I believe @elsassph also had mentioned they use it like that in their code successfully. 

However, I agree, we should refactor `someComponent.callfunc("doThing", invalid)` to `someComponent@.doThing()` as well, so I'll update this PR to support that as well.

Reply:
`callfunc` without argument should be a warning, but I have seen it.

Reply:
yeah, guess you guys are right, if there's a default param on the function, then in that case callfunc without `invalid` will work (I wasn't aware of this or it changed since I got this notion > 2 years ago). Clearly, we can never know if this is the case, so 🤷 I'm not sure if even have a warning here..